python/lsst/ts/electrometer/enums.py

```diff
@@ -25,7 +46,9 @@ class Filter(enum.IntEnum):
 
 class Source(str, enum.Enum):
     IMM = "imm"
+    """Write to buffer immediately"""
     TIM = "tim"
+    """Write to buffer on a timer"""
```

Comment:
Do these doc strings actually show up as documentation in sphinx? I've never managed to make that work, but have heard rumors that it does now.

Please consider inserting a blank line after each doc string (in this case: before TIM) to make it easier to see which element goes with which doc string.